<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreaterMainWindow</class>
 <widget class="QDialog" name="CreaterMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>593</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-29</y>
     <width>178</width>
     <height>552</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ytlogo">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>22</y>
     <width>163</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>20</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
color: white;
background-color: rgb(255, 255, 0, 0);
font-weight: bold;
font-size: 16px;
font-family: PT Sans;
padding-left: 2px;</string>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>22</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
color: white;
background-color: rgb(255, 255, 0, 0);
font-weight: bold;
font-size: 17px;
font-family: PT Sans;
padding-left: 2px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>69</y>
     <width>157</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel, QPushButton {
    border-radius: 11px;
    background-color: black;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    color: white;
    font-size: 17px;
padding-left: -2px;
}

QLabel:hover, QPushButton:hover {
    background-color: red;
    color: white;
}
QLabel:checked, QPushButton:checked {
    background-color: red;
    color: white;
}</string>
   </property>
   <property name="text">
    <string> Upload Video</string>
   </property>
   <property name="icon">
    <iconset resource="Logos.qrc">
     <normaloff>:/Creator's Logo/Logo's/Creator's Logo/Upload Logo.png</normaloff>:/Creator's Logo/Logo's/Creator's Logo/Upload Logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="performance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>157</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel, QPushButton {
    border-radius: 11px;
    background-color: black;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    color: white;
    font-size: 17px;
padding-right: 10px;
padding-left: 3px;

}

QLabel:hover, QPushButton:hover {
    background-color: red;
    color: white;
}

QLabel:checked, QPushButton:checked {
    background-color: red;
    color: white;
}
</string>
   </property>
   <property name="text">
    <string> Performance</string>
   </property>
   <property name="icon">
    <iconset resource="Logos.qrc">
     <normaloff>:/Creator's Logo/Logo's/Creator's Logo/Performance.png</normaloff>:/Creator's Logo/Logo's/Creator's Logo/Performance.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>22</height>
    </size>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>157</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel, QPushButton {
    border-radius: 11px;
    background-color: black;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    color: white;
    font-size: 17px;
padding-left: -42px;
}

QLabel:hover, QPushButton:hover {
    background-color: red;
    color: white;
}
QLabel:checked, QPushButton:checked {
    background-color: red;
    color: white;
}</string>
   </property>
   <property name="text">
    <string>  Logout</string>
   </property>
   <property name="icon">
    <iconset resource="Logos.qrc">
     <normaloff>:/User Logo/Logo's/main window/logouut.png</normaloff>:/User Logo/Logo's/main window/logouut.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>157</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel, QPushButton {
    border-radius: 11px;
    background-color: black;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    color: white;
    font-size: 15px;
padding-left: -3px;
}

QLabel:hover, QPushButton:hover {
    background-color: red;
    color: white;
}
QLabel:checked, QPushButton:checked {
    background-color: red;
    color: white;
}</string>
   </property>
   <property name="text">
    <string> Delete Account</string>
   </property>
   <property name="icon">
    <iconset resource="Logos.qrc">
     <normaloff>:/User Logo/Logo's/main window/delete account.png</normaloff>:/User Logo/Logo's/main window/delete account.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>60</y>
     <width>485</width>
     <height>442</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="upload_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>29</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Browse">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>300</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel, QPushButton {
    border-radius: 5px;
border: 2px solid  rgb(135, 135, 135);
    background-color: white;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    color: rgb(135, 135, 135);
    font-size: 17px;
}
</string>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>270</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;
background-color: rgb(255, 255, 255);
font-family: PTSans;
font-weight: bold;
color: rgb(135, 135, 135);
</string>
     </property>
     <property name="text">
      <string>Upload Video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>380</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
border-radius: 12px;
color: white;
font-size:16px;
font-weight: bold;
background-color: red;</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="videotitle">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 17px;
background-color: #D9D9D9;
color: (127, 127, 127, 0.83);
font-weight: bold;
font-size: 20px;
font-family: PT Sans;
padding-left: 20px;</string>
     </property>
     <property name="placeholderText">
      <string>Enter video title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="videodes">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 17px;
background-color: #D9D9D9;
color: (127, 127, 127, 0.83);
font-weight: bold;
font-size: 20px;
font-family: PT Sans;
padding-left: 20px;</string>
     </property>
     <property name="placeholderText">
      <string>write video Description</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>170</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: #333333;
    background-color: #F5F5F5;
    border: 1px solid #CCCCCC;
    padding: 6px 24px 6px 8px;
    border-radius: 8px;
    selection-background-color: #E0E0E0;
}

QComboBox::drop-down {
    width: 20px;
    border: none;
    background-color: transparent;
}
QComboBox::down-arrow {
    subcontrol-origin: padding;
    subcontrol-position: center right;
    image: url(:/images/dropdown_arrow.png); 
}

QComboBox::drop-down:hover {
    background-color: #E0E0E0;
}

QComboBox::drop-down:pressed {
    background-color: #D9D9D9;
}

QComboBox::down-arrow {
    subcontrol-origin: padding; 
}

QComboBox::item {
    background-color: #F5F5F5;
    color: #333333;
    padding: 4px 8px;
}

QComboBox::item:selected {
    background-color: #E0E0E0;
}
</string>
     </property>
     <property name="placeholderText">
      <string>Select Categorey</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>235</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    spacing: 5px;
    font-size: 14px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff;
    border: 2px solid #999999;
}

QCheckBox::indicator:checked {
    background-color: #007bff;
    border: 2px solid #007bff;
}

QCheckBox::indicator:checked:hover {
    background-color: #0056b3;
    border: 2px solid #0056b3;
}

QCheckBox::indicator:checked:pressed {
    background-color: #003f7f;
    border: 2px solid #003f7f;
}

QCheckBox::indicator:checked {
    font-weight: bold;
}
</string>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>302</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel, QPushButton {
    border-radius: 5px;
border: 2px solid  rgb(135, 135, 135);
    background-color: white;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    color: rgb(135, 135, 135);
    font-size: 17px;
}
</string>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 17px;
font-family: PTSans;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Select video Thumbnail</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 17px;
font-family: PTSans;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Is this Video Age Restricted</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>390</y>
       <width>73</width>
       <height>27</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
border-radius: 12px;
color: white;
font-size:16px;
font-weight: bold;
background-color: red;</string>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Revenue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="revenue">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>157</width>
       <height>35</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel, QPushButton {
    border-radius: 11px;
    background-color: black;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    color: white;
    font-size: 17px;
padding-right: 10px;
padding-left: -35px;

}

QLabel:hover, QPushButton:hover {
    background-color: red;
    color: white;
}

QLabel:checked, QPushButton:checked {
    background-color: red;
    color: white;
}
</string>
     </property>
     <property name="text">
      <string> Revenue</string>
     </property>
     <property name="icon">
      <iconset resource="Logos.qrc">
       <normaloff>:/Creator's Logo/Logo's/Creator's Logo/Revenue.png</normaloff>:/Creator's Logo/Logo's/Creator's Logo/Revenue.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
font-family: PTSans;
font-weight: semi-bold;</string>
     </property>
     <property name="text">
      <string>Video Views: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
font-family: PTSans;
font-weight: semi-bold;</string>
     </property>
     <property name="text">
      <string>Video Likes: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
font-family: PTSans;
font-weight: semi-bold;</string>
     </property>
     <property name="text">
      <string>Video Revenue: </string>
     </property>
    </widget>
    <widget class="QLabel" name="displayviews">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>72</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
font-family: PTSans;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="displaylikes">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>167</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
font-family: PTSans;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="displayrevenue">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>253</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
font-family: PTSans;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>22</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
font-family: PTSans;
font-weight: semi-bold;</string>
     </property>
     <property name="text">
      <string>Sure you want to Logout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>106</width>
       <height>34</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 15px;
    color: white;
    font-size: 19px;
    font-weight: bold;
    background-color: red;
}

QPushButton:hover {
    background-color: #8B0000; 
}
</string>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>12</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
font-family: PTSans;
font-weight: semi-bold;</string>
     </property>
     <property name="text">
      <string>Sure you want to delete your </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dpassword">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>404</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 17px;
background-color: #D9D9D9;
color: (127, 127, 127, 0.83);
font-weight: bold;
font-size: 20px;
font-family: PT Sans;
padding-left: 20px;</string>
     </property>
     <property name="placeholderText">
      <string>Enter Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
font-family: PTSans;
font-weight: semi-bold;</string>
     </property>
     <property name="text">
      <string>account ?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>106</width>
       <height>34</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 15px;
    color: white;
    font-size: 19px;
    font-weight: bold;
    background-color: red;
}

QPushButton:hover {
    background-color: #8B0000; 
}
</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="invalidpassword">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
font-size: 14px;
font-weight: bold;
border-radius: 0px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="deleteresult">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>421</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 27px;
font-family: PTSans;
font-weight: bold;
color: red;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Logos.qrc"/>
 </resources>
 <connections/>
</ui>
